rts/Game/Camera/SpringController.cpp

```diff
@@ -24,7 +25,16 @@ CONFIG(bool,  CamSpringZoomOutFromMousePos).defaultValue(false);
 CONFIG(bool,  CamSpringEdgeRotate).defaultValue(false).description("Rotate camera when cursor touches screen borders.");
 CONFIG(float, CamSpringFastScaleMouseMove).defaultValue(3.0f / 10.0f).description("Scaling for CameraMoveFastMult in spring camera mode while moving mouse.");
 CONFIG(float, CamSpringFastScaleMousewheelMove).defaultValue(2.0f / 10.0f).description("Scaling for CameraMoveFastMult in spring camera mode while scrolling with mouse.");
+CONFIG(bool, CamSpringTrackMapHeight).defaultValue(true).description("Camera height is influenced by terrain height");
```

Comment:
Perhaps I'd rename it to something that indicates the height tracking and make it an int, e.g. if in the future tracking constant height instead of airmesh if desired etc.